### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN029.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#75] [General]
# https://blog.3place.cn/3place/lmyn/LMYN029.html

https://blog.3place.cn/3place/lmyn/LMYN029.html

**Comment by picklecai** [comment]
这苦行者在这里完成道术，
它就以‘完成’这名远近传闻。 （1.28.2）

除了谶语这个词，我想不出别的词来表达这个用法。

**Comment by picklecai** [comment]
请你施展出你那幻术，
请你做一件最大的好事，
你要托生成一个侏儒。 （1.28.7）

这个叫作”完成“的净修林，
就将作为礼物送给你。

懂了，高贵的矮子，是因为毗湿奴托生伟一个侏儒。

**Comment by picklecai** [comment]
于是光辉的毗湿奴，
就托生在阿底提胎里，
变成一个侏儒的样子，
走向毗卢遮那之子婆离。 （1.28.9）

他要求三步能跨过的土地，
他得到了婆离的许诺，
为了一切众生的幸福，
世界之魂把众世界跨过〔284〕。 （1.28.10）

之所以要变成侏儒，是因为后面要做这个赌约：三步能跨过的土地。
在常人看来，一个小矮子，三步能跨多少？送就送吧。
于是赌约成立。
于是这家伙把人家一整个国家都跨过了。
这个故事想来很有意思。

**Comment by picklecai** [comment]
收视返听。
收回向外看的视线，向外听的耳朵。专注于自身。说内观也好，说放松也好，说发呆也好。放空，以真正的休息。 ​​​
